### ISSUE TITLE
Issue at 2d

### ISSUE THREAD

**Issue by grace-ac** [RobertsLab/project-oyster-oa#27]
So, everything has worked great- I had no issue finding the RAW files in the downloads "Archive" folder today.

However, I'm at Step 3b: Clean Dilution Curve Data, and I don't know what is wrong. All the windows are blank...?
![capture1](https://user-images.githubusercontent.com/14934314/31522533-1149f0a4-af63-11e7-9f2e-fc762c257a16.PNG)

**Comment by yaaminiv** [comment]
Opp, I think you're following an old version of the protocol! Repeat Step 2d (I revised it yesterday so you wouldn't encounter your specific issue).

**Comment by yaaminiv** [comment]
@grace-ac Did that ^ work?

**Comment by grace-ac** [comment]
I haven't tried it yet - I'll be back in lab on Monday afternoon to test it out! sorry if you're waiting on it :(

**Comment by yaaminiv** [comment]
No worries! Just wanted to make sure I wasn't missing anything.

**Comment by grace-ac** [comment]
This happened after re-doing 2d:

![capture](https://user-images.githubusercontent.com/14934314/31632389-9d39c754-b271-11e7-9e4b-e2474f657c41.PNG)

**Comment by yaaminiv** [comment]
That's odd! Didn't happen to me on the same machine you're using when I tried it out. What happens when you press cancel? I don't understand what error it's seeing. Any suggestions @emmats ?

**Comment by grace-ac** [comment]
I just pressed cancel, and my screen looks exactly how it does in the original screenshot of this issue

**Comment by yaaminiv** [comment]
Huh...what does it look like when you press OK?

**Comment by grace-ac** [comment]
![capture](https://user-images.githubusercontent.com/14934314/31632909-79acc3ca-b273-11e7-9b78-7ea1aabf0c69.PNG)

**Comment by yaaminiv** [comment]
It seems like you kept all of the previously imported transitions before you redid 2d! Try deleting absolutely everything from that lefthand window and re-importing the transition list.

**Comment by emmats** [comment]
Let me know if you still need my input.

**Comment by grace-ac** [comment]
I restarted from the beginning and still got the error. 

Things look much better though, after clicking, "OK" and I'm currently working on Step 3b.

**Comment by yaaminiv** [comment]
@emmats do you know what could be causing this error? I exported the transition list from Skyline itself.

@grace-ac You should open up the transition list and make sure the transitions in your Skyline document are the same ones in the transition list you're importing.

**Comment by grace-ac** [comment]
I went through and I matched the numbers on the list to those in Skyline doc. I just highlighted the first three to show what numbers I was matching. 
![capture](https://user-images.githubusercontent.com/14934314/31635605-2c991d8c-b27c-11e7-9f71-4d4fab1affac.PNG)
![capture1](https://user-images.githubusercontent.com/14934314/31635606-2cae65ac-b27c-11e7-9408-f3d859b5d955.PNG)

All of the ones in Skyline Document had a match in the original document. The only ones that weren't in the Skyline doc were the ones that were labeled "peptides" rather than a number. 
![capture2](https://user-images.githubusercontent.com/14934314/31635625-442c56d0-b27c-11e7-9abf-91aca9b96c76.PNG)

**Comment by yaaminiv** [comment]
AH those are the PRTC peptides! I think I know what's causing the error so I can fix that for you. It won't affect any of your downstream analyses.

You can also click on the plus signs to expand the protein and see the peptides/transitions associated with it

**Comment by grace-ac** [comment]
Okay - I clicked the plus signs and they all match! woo!

Let me know when you fix whatever is causing the error and i'll re-do it!

**Comment by yaaminiv** [comment]
In the transition list, what would happen if you removed the space between PRTC and peptides, or got rid of that phrase entirely (keeping the other content in that row)?

**Comment by grace-ac** [comment]
I tried both suggestions, and both resulted in the same error.

**Comment by emmats** [comment]
I've never seen this error before so I can't really weigh in on what is going wrong. If you can't figure it out, I can probably stop by tomorrow afternoon or Monday.

**Comment by grace-ac** [comment]
I realize that you've been gone @yaaminiv , but I tried it again, because earlier in this thread you said you were going to fix something - thought I'd see if anything has changed.

Error happened again.
![capture](https://user-images.githubusercontent.com/14934314/31971372-67862142-b8d0-11e7-93c8-7f38d954e878.PNG)

Would it have anything to do with the fact that I'm using Excel? 
![capture-o1](https://user-images.githubusercontent.com/14934314/31971287-fe451b70-b8cf-11e7-8c89-e04080af08a5.PNG)

**Comment by kubu4** [comment]
What is the full error message from the screenshot @grace-ac posted in this comment: https://github.com/RobertsLab/project-oyster-oa/issues/27#issuecomment-337018610

**Comment by grace-ac** [comment]
I just updated my comment @kubu4 
is that what you are looking for?

**Comment by kubu4** [comment]
Nope. The error messages are truncated and can't be read. Need to be able to read the entire message for each line.

**Comment by yaaminiv** [comment]
I don't think it's an Excel issue! I think it's some sort of problem with Skyline where it can't recognize the peptides from the transition list because the retention times in the transition list do not align with what's in the library...? I'm still working on it, but this issue won't affect anything in the SRM protocol since I haven't dealt with my PRTC peptides yet.

**Comment by grace-ac** [comment]
That's as much as I could get. 
![capture](https://user-images.githubusercontent.com/14934314/31971642-7bfd6314-b8d1-11e7-9b06-c9ad76658723.PNG)

Not really sure I understand what's happening - whether I should continue with the protocol or wait for you to investigate things @yaaminiv

**Comment by yaaminiv** [comment]
@grace-ac Continue!! It's possibly over my head as well so don't wait for me.

**Comment by kubu4** [comment]
For kicks @grace-ac , when that line is highlighted, can you copy it (right-click or ctrl-c), and then paste in a text editor for viewing?

**Comment by grace-ac** [comment]
@kubu4 No, but I could hold the mouse over and read the remaining words:
"match the tolerance on the Instrument tab of the Transition Settings."

Same for all of them.

**Comment by yaaminiv** [comment]
@grace-ac Hold up! Continue with 2e, then retry 2d. I have a hunch that after changing your transition settings, you'll be able to import everything

**Comment by grace-ac** [comment]
Same error. The transition settings were already set according to your protocol from my previous attempts

**Comment by yaaminiv** [comment]
Darn. I'm not sure what it is so I'll play around with it tomorrow, but again, it won't affect anything else in that protocol

**Comment by grace-ac** [comment]
Cool beans - I shall continue!

**Comment by grace-ac** [comment]
3b is where I'm stuck - which is where I believe I started this whole issue in the first place. 
![capture](https://user-images.githubusercontent.com/14934314/31972528-001724ba-b8d6-11e7-87ef-80ae3fd9c462.PNG)

Does not look like your screenshot for Step 3b:
![capture1](https://user-images.githubusercontent.com/14934314/31972579-3b3dbe14-b8d6-11e7-8bc7-52688bcd36fd.PNG)

**Comment by yaaminiv** [comment]
Ah! Instead of clicking on the protein name, you want to click on the peptide (i.e. the things with the green dots next to them)

**Comment by grace-ac** [comment]
So, am I comparing the peptide retention time to the average SRM retention time? 
![capture](https://user-images.githubusercontent.com/14934314/32012659-75f374ee-b96d-11e7-82dd-f565969db4d5.PNG)

**Comment by yaaminiv** [comment]
Yup! You compare the average retention time with the retention times in the Skyline document. The correct peak will have all three transition fragments.

**Comment by yaaminiv** [comment]
@grace-ac Do you think you can create a new issue with the Skyline error message you keep getting? I want to address the other questions in this issue and close it out, then refer emma to the new issue you'll create!

**Comment by yaaminiv** [comment]
Resolved issues at 3b, but not the import issue at 2d. @grace-ac will create a new issue and I will continue the discussion there.
